canonical-web-and-design/vanilla-framework
# Update component status 2.5.0

## Done

Removed old status labels from component docs pages.
Removed unnecessary menu-button from docs.

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/
- Check accordion, pagination, code, sub navigation docs - there should be no status labels on them anymore
